### PR TITLE
updated .gitignore

### PR DESCRIPTION
**Pull request by adparvum**
@majuscule

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,21 +1,104 @@
...
+__pycache__/
+*.py[cod]
 *$py.class
+
+# C extensions
 *.so
-__pycache__
-.coverage
-.installed.cfg
-.mr.developer.cfg
```
Comment (majuscule) [inline .gitignore:9]: i think this one should be copied over

Comment (majuscule) [reply .gitignore:9]: add it to the botom with a comment `# mr.developer` (it's apparently part of this: https://pypi.python.org/pypi/mr.developer

**Review by majuscule — CHANGES_REQUESTED** [review]
LGTMBC

File: .gitignore [hunk 2]
```diff
@@ -1,21 +1,108 @@
...
+# Rope project settings
+.ropeproject
+
+# mkdocs documentation
+/site
+
+# mypy
+.mypy_cache/
+
+# dataflake.fakeldap
```
Comment (majuscule) [inline .gitignore]: these comments are referring to what generates the files. *everything* here is for `dataflake.fakeldap`. [mr.developer](https://pypi.python.org/pypi/mr.developer) is what generates `.mr.developer.cfg`, and that is what the section header should be.

**Review by majuscule — APPROVED** [review]
LGTM. You should make this PR upstream as well and then include the link here.

**Comment by adparvum** [comment]
made upstream PR https://github.com/dataflake/dataflake.fakeldap/pull/5/

**Pull request merged**